bloom/generators/debian/templates/rules.em

```diff
@@ -59,4 +60,7 @@ override_dh_auto_install:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
-	dh_auto_install
+	dh_auto_install --destdir=$(CURDIR)/debian/@(Package)
```

Comment:
Ok, I'm most uncertain/concerned about this part of the change because I'm unfamiliar with the underlying mechanism, but we can possibly test it out to make sure it doesn't break anything.